src/main/kotlin/lotto/domain/MatchingMachine.kt

```diff
@@ -0,0 +1,18 @@
+package lotto.domain
+
+object MatchingMachine {
+
+    fun match(
+        lotto: Set<LottoNumber>,
```

Comment:
이왕 만든 `LottoTicket`을 사용하면 어떨까요?

```suggestion
        lotto: LottoTicket,
```